Common/Statistics/PortfolioStatistics.cs

```diff
@@ -235,6 +236,14 @@ public static decimal GetRiskFreeRate()
             return RiskFreeRate;
         }
 
+        /// <summary>
+        /// Gets the current defined maximum tradeable volume
+        /// </summary>
+        public static decimal GetMaximumTradeableVolume()
+        {
+            return MaximumTradeableVolume;
```

Comment:
If we're going to make the value a constant _and then_ provide a static function to return the constant, why not just make the constant directly public? I could see if this value was settable and/or overridable via a subclass or similar, but as written, it seems like an extra level of indirection for no benefit